### PR TITLE
ModelMesh.Tag fix

### PR DESCRIPTION
**Pull request by AndersonReyes**
Fixed issue where the ModelMesh.Tag instance was being ignored issue #3649

### CODE REVIEW

File: MonoGame.Framework/Content/ContentReaders/ModelReader.cs [hunk 1]
```diff
@@ -149,9 +149,6 @@ protected internal override Model Read(ContentReader reader, Model existingInsta
                 var parentBoneIndex = ReadBoneReference(reader, boneCount);
 				var boundingSphere = reader.ReadBoundingSphere();
 
-                // Tag
-                reader.ReadObject<object>();
-
```
Comment (tomspilman) [inline MonoGame.Framework/Content/ContentReaders/ModelReader.cs]: This looks broken to me.  You can't move the `ReadObject` call...  that has to remain where it is to obey the file format for models.

Did you test it with your own game?

Comment (AndersonReyes) [inline MonoGame.Framework/Content/ContentReaders/ModelReader.cs]: But there is no mesh from ModelMesh class instance before that which is what the code has to write ain't it?

Comment (tomspilman) [inline MonoGame.Framework/Content/ContentReaders/ModelReader.cs]:
```
var meshTag = reader.ReadObject<object>();
```

Then assign it below.

Comment (AndersonReyes) [inline MonoGame.Framework/Content/ContentReaders/ModelReader.cs]: then assigned it below like mesh.Tag = meshTag?

Comment (tomspilman) [inline MonoGame.Framework/Content/ContentReaders/ModelReader.cs]: Yes.

Comment (lllAlexlll) [inline MonoGame.Framework/Content/ContentReaders/ModelReader.cs]: Tested. I confirm that the issue is fixed now.

**Pull request closed without merging**